JavaScript Proxy SDK/README.MD

```diff
@@ -0,0 +1,158 @@
+# JavaScript Proxy SDK example
+Create a feature and connect it to client app using the JavaScript Proxy SDK and at the end we play around with some strategies.
+
+## Let's get started
+1. Create a folder then navigate to the folder through your terminal and type:  
+`npm init -y`  
+This should create a file called _package.json_.
+
+2. Create two files:
+
+* index.html
+* app.js
+
+inside _index.html_ paste the code below
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <script src="app.js" type="text/javascript"></script>
+    <title>Unleash Testing</title>
+  </head>
+  <body>
+    <h1>Hello from Unleash Team !</h1>
+  </body>
+</html>
+```
+
+If you open your _index.html_ you should see something like this 👇
+<br/>
+<img src="./static/img/hello-unleash.png" title="index page" />
+
+Now let's install the JavaScript Proxy SDK, run:
+
+```sh
+npm install unleash-proxy-client
+```
+
+Let's import the JavaScript Proxy SDK to our html file:
+
+```html
+<script
+  src="../node_modules/unleash-proxy-client/build/main.min.js"
+  type="text/javascript"
+></script>
+```
+
+Now let initialize the SDK
+
+```js
+let config = {
+  url: "https://app.unleash-hosted.com/demo/proxy",
+  clientKey: "proxy-123",
+  refreshInterval: 1, // 1 second
+  appName: "my-webapp",
+};
+
+let client = new unleash.UnleashClient(config);
+client.updateContext({ userId: "123" }); // will be used later
+client.start();
+
+console.log(client.isEnabled("my-awesome-feature"));
+
+// this code will log every second the state of our feature
+setInterval(() => {
+  console.log(client.isEnabled("my-awesome-feature"));
+}, 1000);
+```
+
+**Note ⚠️:** _my-awesome-feature_ is my feature name. You have to change it with your feature name that you have created. [Don't know how to create a feature ?](https://docs.getunleash.io/user_guide/create_feature_toggle)
+
+Now if we refresh the _index.html_ page and open the console we should see something like this which is the feature state _(depends on your toggle state)_ 👇 
+<br/>
+<img src="./static/img/feature-on-console.png" title="Login page Unleash" /> 
+Note that when we disabled our feature we got _false_ printed in the console.
+
+
+Now we want to change the background color of our _index.html_ file to _red_ when the feature is on and to _green_ when the feature is off.
+
+To do that we need a function that checks if the state of our feature change to run a some code. Luckly we have the function we need in Unleash JavaScript Proxy SDK 👇
```

Comment:
⚠️  It is important to note that any event listeners must be registered before the user call `client.start().` If not you may loose events because they can be triggered before you have a chance to register your event listeners.